README.md

```diff
@@ -4,13 +4,22 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+# About the Project
+**UGTeach** empowers undergraduate private tutors to efficiently manage payments, track student performance, and organize schedules. It streamlines tutoring operations, ensuring you stay organized and focused on helping your students achieve their academic goals.<br>
+
+  **Example usages:**
+  * As a private tutor, you can add student's personal information and their tuition details
+  * As a private tutor, you can view the list of students you are currently tutoring
+  * As a private tutor, you can view the amount of money you have earned from tutoring
+
+# More Information
+  * It is **written in OOP fashion**. 
+  * It is **written in Java**, and has about 6 kLoC.
```

Comment:
6K LoC instead of 6 kLoC

Reply:
I have changed `6 kLoC` to `6 KLoC`. The template given to us initially was 6 KLoC. Would this be ok? @Soumyadip-cmd

Reply:
Ok then LGTM